### ISSUE TITLE
Quick tags buttons need to be properly labeled (regression from current editor)

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#979]
In the current WordPress text editor, the QuickTags buttons are properly labeled (the labels are also translatable):

![screen shot 2017-06-01 at 18 31 47](https://cloud.githubusercontent.com/assets/1682452/26690272/e94bb484-46f8-11e7-82bd-a5dd7dcce040.png)

In Gutenberg, they're not (yet?):

![screen shot 2017-06-01 at 18 31 28](https://cloud.githubusercontent.com/assets/1682452/26690278/f3ebec74-46f8-11e7-938c-c6174c74e7b4.png)

The current rendered markup is, for example:
`<button class="editor-text-editor__bold">b</button>`

All these buttons need `aria-label` attributes with translatable strings, as in the current text editor.
To evaluate: do they need also a `type="button"` attribute?

Aside: as mentioned in another issue, personally I'm not fully convinced QuickTags should stay but if they have to stay they should be accessible and not introduce accessibility regressions.

**Comment by afercia** [comment]
See #761

**Comment by jasmussen** [comment]
Given I just closed the quick tags ticket, I'm also going to take the liberty of closing this one. That can let us focus all the discussion around how to best handle #1733. If we need to — say we decide to include both codemirror and a plain textfield, we can reopen the quicktags tickets.
